packages/integrations/package.json

```diff
@@ -16,13 +16,15 @@
   "module": "esm/index.js",
   "types": "dist/index.d.ts",
   "dependencies": {
-    "@sentry/types": "5.22.3",
-    "@sentry/utils": "5.22.3",
+    "@sentry/browser": "^5.22.3",
+    "@sentry/types": "5.22.2",
+    "@sentry/utils": "5.22.2",
     "localforage": "1.8.1",
     "tslib": "^1.9.3"
   },
   "devDependencies": {
-    "@sentry-internal/eslint-config-sdk": "5.22.3",
+    "@sentry-internal/eslint-config-sdk": "5.22.2",
+    "@types/uuid": "^8.3.0",
```

Comment:
Remove dependency

Reply:
removed "@types/uuid": "^8.3.0"  and revert @sentry-internal/eslint-config-sdk module version